src/Analyzers/CSharp/CodeFixes/UseUTF8StringLiteral/UseUTF8StringLiteralCodeFixProvider.cs

```diff
@@ -175,21 +175,26 @@ private static SyntaxNode CreateArgumentListWithUTF8String(BaseArgumentListSynta
             return argumentList.WithArguments(SyntaxFactory.SeparatedList<ArgumentSyntax>(arguments));
         }
 
-        private static LiteralExpressionSyntax CreateUTF8String(SyntaxNode nodeToTakeTriviaFrom, string stringValue)
+        private static InvocationExpressionSyntax CreateUTF8String(SyntaxNode nodeToTakeTriviaFrom, string stringValue)
         {
             return CreateUTF8String(nodeToTakeTriviaFrom.GetLeadingTrivia(), stringValue, nodeToTakeTriviaFrom.GetTrailingTrivia());
```

Comment:
```suggestion
            return CreateUTF8String(stringValue).WithTriviaFrom(nodeToTakeTrivia);
```

Reply:
As per below, switching this to use `WithTriviaFrom` would work, but the method would have to pass two empty trivia lists for the leading and trailing trivia arguments, so not worth it.

Reply:
Thanks!!